<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1205</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>1161</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>체결정보</string>
     </attribute>
     <widget class="QPushButton" name="run_btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>실 행</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditForRun">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="concludeByItematDate">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>70</y>
        <width>161</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>체결정보그래프보기</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>126</width>
        <height>25</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="currentSection">
       <enum>QDateTimeEdit::DaySection</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="item_comboBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>161</width>
        <height>24</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="maxVisibleItems">
       <number>15</number>
      </property>
     </widget>
     <widget class="QTableWidget" name="concTableWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>531</width>
        <height>431</height>
       </rect>
      </property>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>23</number>
      </attribute>
     </widget>
     <widget class="QPushButton" name="reArrange">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>125</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>재정렬</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>130</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>원 보다 큰 거래 합</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_second">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>90</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>130</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>거래대금이</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_TransVolume">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>130</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>100000000</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>90</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>초 동안(0이면 체결마다)</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="concTableWidget_2">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>160</y>
        <width>521</width>
        <height>401</height>
       </rect>
      </property>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>23</number>
      </attribute>
     </widget>
     <widget class="QPushButton" name="concluionDetail">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>124</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>그래프</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>90</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>장중</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_TransVolume_sum">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>130</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="dateLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>대량거래추이</string>
     </attribute>
     <widget class="QDateEdit" name="dateEdit_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>75</y>
        <width>126</width>
        <height>25</height>
       </rect>
      </property>
      <property name="currentSection">
       <enum>QDateTimeEdit::DaySection</enum>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_3">
      <property name="geometry">
       <rect>
        <x>233</x>
        <y>75</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="currentSection">
       <enum>QDateTimeEdit::DaySection</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>79</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>~</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>종목명</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditForRun_2">
      <property name="geometry">
       <rect>
        <x>93</x>
        <y>40</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>40</y>
        <width>75</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>기간</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>거래금액</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditForRun_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>116</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>1071</width>
        <height>371</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="ConcHistoryGraph"/>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1205</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>파일</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>도구</string>
    </property>
    <addaction name="call_Itemsort"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>옵션</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="call_Itemsort">
   <property name="text">
    <string>관심종목 추가/삭제</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
